SparkPost/2web2ui
# SA-1431- Create Cypress Tests for Report Builder

[SA-1431](https://jira.int.messagesystems.com/browse/SA-1431)

### What Changed
- Added Cypress tests for 'Report Builder' ('Analytics Report'? Should the files be renamed?)
- Cypress-produced screenshots are now uploaded to CircleCI's storage when errors occur in integration tests
- Introduced a few style tweaks here and there where necessary
- Added a few `data-id` attributes for tests

### How To Test
- Run the tests locally and be sure they pass:
  + `npm run start-app-hibana` in one terminal window/tab
  + `npm run test-integration-hibana` in one terminal window/tab
- Go through tests and verify relevant scenarios are captured by the automated testing
- I am _sure_ there are scenarios I've missed here

### To Do
